pkg/operator/sync.go

```diff
@@ -1,70 +1,74 @@
 package operator
 
 import (
-	"fmt"
-
+	"github.com/golang/glog"
 	operatorv1 "github.com/openshift/api/operator/v1"
 )
 
 func syncControllers(c serviceCAOperator, operatorConfig *operatorv1.ServiceCA) error {
+	// Any modification of resource we want to trickle down to force deploy all of the controllers.
 	// Sync the controller NS and the other resources. These should be mostly static.
-	modified, err := manageControllerNS(c)
+	needsDeploy, err := manageControllerNS(c)
 	if err != nil {
-		return fmt.Errorf("Error syncing NS: %v", err)
+		return err
```

Comment:
Are the respective `Apply*()` functions verbose enough (and yet not too much), that we'll be able to track the errors to the specific controllers?

Reply:
Yes, I think so. We will get the kapi errors that tell us resource type and name.